internal/platform/logger/spy/formatter.go

```diff
@@ -0,0 +1,56 @@
+package spy
```

Comment:
if we don't have tests, do we need this package?

Reply:
deleted